coverage/.resultset.json

```diff
@@ -1,51 +1,57 @@
 {
```

Comment:
this file should not be here

Reply:
I have added the coverage to the gitignore, which was not done before, I do not know how to remove these files now...